benchmark/sirun/run-all-variants.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
```

Comment:
Is there a plan to move all this logic to sirun since it will need to be duplicated in all languages?

Reply:
Because for some folks/use-cases it might make sense to tear down an environment completely between variants (e.g. start a new container, etc.), it seemed to make more sense to just have sirun run one variant at a time, and leave any orchestration of that up to users.

Reply:
On second thought, there's definitely room for at least an _option_ in sirun to run all the variants. I'll make that change in sirun when I have some time. In the meantime, we'll keep this script around. Sound good?